### ISSUE TITLE
Getting error code as "network"

### ISSUE THREAD

**Issue by jmriyaz84** [mdn/web-speech-api#58]
I have implemented Speech Recognition in my project , it is working fine in my home network , but in office network I am getting error code as "network" , may be some firewall is blocking .

I can ask my network team to unblock but I want to know internally this API is connecting this which URL , so that I can ask my network team to whitelist that URL . 

Please share me the URL which is converting Speech to Text & Text to Speech in this API.

**Comment by clemens-tolboom** [comment]
I guess this is a support question better suited on stackexchange but I try to answer.

> it is working fine in my home network

What URLs are you using?

My guess is `http` and localhost / 127.0.0.1 and not `https`?

At least I ran into something like that. Even creating a selfsigned certificate to make it work fails with current browsers state (https only/etc).

**Comment by schalkneethling** [comment]
I agree with @clemens-tolboom that this would be better suited to StackOverflow. Also, note that we are combining some of our repositories to enable us to triage pull requests and issues more effectively. Therefore, this specific repository will be merged into the more prominent [dom-examples](https://github.com/mdn/dom-examples) repository.
